cmd/podman/image.go

```diff
@@ -14,14 +14,14 @@ var (
 			Long:  imageDescription,
 		},
 	}
+	_imagesSubCommand = _imagesCommand
```

Comment:
Does this actually copy it? I'm wondering how we get two images commands with different aliases out of a simple equals

Reply:
Alright, so we're doing actual values here, instead of pointers, and that forces the copy. Ok...

Reply:
we do not ... at least when the original was defined as a pointer.  this was the only way it actually worked.

Reply:
Yeah, it looks like Go automagically copies when it's a struct assigned by value, not reference.

Reply:
Can we have a comment here like "Don't use pointer to force copy" so someone doesn't see this in a month and say "why is this different? I'm going to make this look like the others"